### ISSUE TITLE
View List of Letters - accessibility issues

### ISSUE THREAD

**Issue by digitalken** [department-of-veterans-affairs/va.gov-team#16252]
## What happened?
~~1. On Android with bluetooth keyboard, using the up/down arrow skips over large number of menu items, "back" button, and page title ("Letters").  It skips the header items (back and title) then gets to "Proof of Service Letter".  I then must click the down arrow many times before it finally moves, but it skips to "Benefit summary Letter".  The other menu items between are skipped over.~~

2. Tabbing seems to tab through menu items correctly along with the bottom nav bar but doesn't seem to work in header correctly.  I have to tab many times to get it to move off of the "back" button, and when it does, it's skips the title header ("Letters").

3. With the appropriate Screen Reader enabled on iOS and android, the back button a11yHint is not being announced
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

## Specs:
- Device: Motorola Moto 7 with bluetooth keyboard linked.
- Device: iPhone XR with VoiceOver enabled 
- Device: LG LM-Q720 Stylo 5 with TalkBack enabled

## Steps to Reproduce

~~For number 1:~~
~~Navigate to Letters page and attempt to use up/down arrows to traverse page.
Tap up or down arrows until you start to traverse the page.
When you get to the top or bottom, tap the other arrow to go the other way.
Notice it skipping over many menu item buttons.~~

For number 2:
- Navigate to the Letters page and tap the Tab button 8 or 9 times before the "Back" button is highlighted.
- Tab again until another item is highlight.  It should skip "Letters" and highlight the first menu item, "Commissary Letter".
- Tab more and you'll find that it traverses the page properly clear through the nav bar buttons.

## Desired behavior
~~For number 1:
Using the up / down arrows, user should be able to traverse the page in order and stop on every menu item as well as the back button and the title header.~~

For number 2:
- Using the tab button, all elements of the page should be traversed.

## Acceptance Criteria
For numbers 1 & 2:
- All elements of the page may be traversed using the tab button or the up / down arrow keys.

For number 3:
- The back button a11yHint will be read out as “Go back to the letters landing page”

## Bug Severity
- 4 - low

## Linked to Story
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/13941

## Screen shot(s) and additional information

**Comment by bessgreen** [comment]
Update: navigating via up/down arrows on keyboard is not required for accessibility: navigating via the Tab key is. Omitting item #1 from this story.

**Comment by digitalken** [comment]
PASS 3/4/2021 - Tab goes through all elements of page with Android.  iOS does not navigate to the 'Back' button and is covered in a new bug #20843.  Closing.
